tools/testing/test_selections.py

```diff
@@ -18,7 +18,11 @@
 
 IS_MEM_LEAK_CHECK = os.getenv("PYTORCH_TEST_CUDA_MEM_LEAK_CHECK", "0") == "1"
 BUILD_ENVIRONMENT = os.getenv("BUILD_ENVIRONMENT", "")
-USE_3_PROCS = "sm86" in BUILD_ENVIRONMENT or "cuda" not in BUILD_ENVIRONMENT
+USE_3_PROCS = (
```

Comment:
Is this possible to `import torch` here and just apply this logic https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_cuda.py#L33?